data-raw/rs_onco_pcwg3.R

```diff
@@ -19,7 +19,7 @@ pcwg3_raw <- tibble::tribble(
   "01-701-1034", "WEEK 16",     "PD",        "SD", "NON-PD",     "SD",
   "01-701-1034", "WEEK 24",     "PD",        "SD", "NON-PD",     "SD",
   "01-701-1097",  "WEEK 8",     "PR",        "SD", "NON-PD",     "SD",
-  "01-701-1097", "WEEK 16",     "PR",        "PR",    "PDu",     "PR",
+  "01-701-1097", "WEEK 16",     "PR",        "PR", "NON-PD",     "PR",
```

Comment:
I think this is not correct. As PD requires confirmation, you can't have a single PD. Either it should be PD at WEEK 16 or it should be PDu at WEEK 24.

Do you agree?